filament/src/ColorGrading.cpp

```diff
@@ -54,8 +97,8 @@ Tonemapper::Tonemapper(FEngine& engine) {
     JobSystem& js = engine.getJobSystem();
     auto slices = js.createJob();
     for (size_t b = 0; b < LUT_DIMENSION; b++) {
-        auto job = js.createJob(slices,[data, b](JobSystem&, JobSystem::Job*) {
-            half4* UTILS_RESTRICT p = (half4*)data + b * LUT_DIMENSION * LUT_DIMENSION;
+        auto job = js.createJob(slices, [&, data, b](JobSystem&, JobSystem::Job*) {
```

Comment:
I prefer we list exactly what we need to capture. If you switch to a pointer-to-member it might be possible to capture `tonemapper` by value, which would be better.

Reply:
We go over with an std::function, we should be good with the pointer. Eventually I will need to pass more data so all the inputs will be inside an intermediate struct anyway.